tsconfig.json

```diff
@@ -12,7 +12,8 @@
       ],
       "test_utils/*": [
         "src/test_utils/public/*"
-      ]
+      ],
+      "fixtures/*": ["src/fixtures/*"]
```

Comment:
Do we really need this alias? I don't know how much `fixtures` will be used in the future and IMHO it's better to only introduce these if they are really necessary - just using a relative import seems easier to me. But maybe I'm missing an aspect here.

Reply:
Yeah, I was questioning whether to add this, but the fixtures files have the `fixtures/` aliased paths in them so a relative import still fails. Do you think I should change all the paths in the fixtures files to be relative and get rid of this alias?

Reply:
Ah good point. @elastic/kibana-operations do you have a suggestion?

Reply:
IMHO I think we keep this for now until we phase out the fixtures completely. Mainly because I think it's strange to have a path alias in javascript that is not also in typescript.